src/main/resources/db/migration/V013__Add_new_fields.sql

```diff
@@ -2,17 +2,14 @@ ALTER TABLE envelopes
   ADD COLUMN caseNumber VARCHAR(100) NULL;
 
 ALTER TABLE non_scannable_items
-  ADD COLUMN documentControlNumber VARCHAR(100) NOT NULL;
+  ADD COLUMN documentControlNumber VARCHAR(100) NOT NULL DEFAULT '1111111';
```

Comment:
do we actually want to use this value in the future whenever a record without the `documentControlNumber` is inserted???

Reply:
follow up to remove the default now that its been applied in prod?

Reply:
yes, now that all of this is applied - the follow up migration can be done as well